python-requirements.txt

```diff
@@ -32,3 +34,6 @@ git+https://github.com/lowRISC/fusesoc.git@ot#egg=fusesoc >= 1.11.0
 
 # Development version with OT-specific changes
 git+https://github.com/lowRISC/edalize.git@ot#egg=edalize >= 0.2.0
+
+# Development version of ChipWhisperer toolchain, needs to be installed in editable mode.
+-e git+https://github.com/newaetech/chipwhisperer.git@develop#egg=chipwhisperer
```

Comment:
I didn't find a reason to install chipwhipsperer in editable mode, the loading seems to work without it? If we don't have one, let's avoid it to give us a cleaner installation.

The chipwhisperer git archive is rather large, installing from an archive makes the installation much faster (it's still a 60 MB download!).

```suggestion
# Development version of ChipWhisperer toolchain
# Use a development version until support for the CW310 board works in a
# released version.
# Prefer the archive download over a git clone to decrease installation time
# (The chipwhisperer repository is rather large and uses additionally uses
# submodules, which need to be fetched as well.)
https://github.com/newaetech/chipwhisperer/archive/9b6825b495f14f85aae8b11007c63c59a1654584.tar.gz
```

Reply:
Thanks, this is clearly the better option. I've implemented the change as suggested and also tested it successfully.

About the reason for the editable mode: It's actually recommend by ChipWhisperer: https://chipwhisperer.readthedocs.io/en/latest/installing.html#chipwhisperer

But as the method you suggested works as well, I think it's better to stick to this for the moment.

Reply:
For posterity - the editable mode was recommended as many users are tracking changes in CW rather than using specific releases, and end up with "local changes" themselves. So it was more of a usability feature we found, but forgot to mention the reason in our docs. So will fix that (doesn't affect OT).